Packs/Core/Playbooks/Playbook-Large_Upload.yml

```diff
@@ -0,0 +1,3868 @@
+      {
+        "position": {
+          "x": 450,
+          "y": 140
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 25929536-ed1b-4848-810f-8b2023b1d8fb
+    type: condition
+    task:
+      id: 25929536-ed1b-4848-810f-8b2023b1d8fb
+      version: -1
+      name: Found Results?
+      description: Determine if previous false positive alerts have been detected with similar characteristics.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "6"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: isSimilarIncidentFound
+            iscontext: true
+          right:
+            value:
+              simple: "true"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 300
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: dc4effe6-7a0d-45d2-8301-15c8d4234cd0
+    type: condition
+    task:
+      id: dc4effe6-7a0d-45d2-8301-15c8d4234cd0
+      version: -1
+      name: Investigation Should be Continued?
+      description: Determines whether an alert requires further investigation if similar previous false positive incidents were found.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "7"
+      "yes":
+      - "5"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: inputs.FurtherInvestigation
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1840,
+          "y": 470
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 448fefe4-c69a-45fd-899c-6e138ce31e92
+    type: condition
+    task:
+      id: 448fefe4-c69a-45fd-899c-6e138ce31e92
+      version: -1
+      name: Alert Investigation Should Continue?
+      description: |-
+        The playbook detected similar alerts that were closed as false positives previously.
+
+        Review the alert details and similar closed false positive alerts to determine whether this Cortex XDR alert should be further investigated.
+      type: condition
```

Comment:
maybe we should add here an ask option instead of a manual? like typical "yes"/"no"

Reply:
When I first built the playbook (on XSOAR) I did use the ask option but Yarden asked me to replace it with a manual task to reduce the number of tasks.